From 0488e742dcf68d044910b51beddc712ae1baa767 Mon Sep 17 00:00:00 2001
Message-Id: <0488e742dcf68d044910b51beddc712ae1baa767.1387382496.git.minovotn@redhat.com>
In-Reply-To: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
References: <c5386144fbf09f628148101bc674e2421cdd16e3.1387382496.git.minovotn@redhat.com>
From: Nigel Croxon <ncroxon@redhat.com>
Date: Thu, 14 Nov 2013 22:53:08 +0100
Subject: [PATCH 32/46] rdma: remaining documentation fixes

RH-Author: Nigel Croxon <ncroxon@redhat.com>
Message-id: <1384469598-13137-33-git-send-email-ncroxon@redhat.com>
Patchwork-id: 55710
O-Subject: [RHEL7.0 PATCH 32/42] rdma: remaining documentation fixes
Bugzilla: 1011720
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Bugzilla: 1011720
https://bugzilla.redhat.com/show_bug.cgi?id=1011720

>From commit ID:
commit 8f3067bd86485f8cd03abc940ddb2b8467ef3627
Author: Michael R. Hines <mrhines@us.ibm.com>
Date:   Fri Aug 9 16:05:45 2013 -0400

    rdma: remaining documentation fixes

    Was missing 'setup-time' in some of the QMP documentation...

    Signed-off-by: Michael R. Hines <mrhines@us.ibm.com>
    Reviewed-by: Eric Blake <eblake@redhat.com>
    Message-id: 1376078746-24948-7-git-send-email-mrhines@linux.vnet.ibm.com
    Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
---
 qmp-commands.hx |   10 ++++++++++
 1 files changed, 10 insertions(+), 0 deletions(-)

Signed-off-by: Michal Novotny <minovotn@redhat.com>
---
 qmp-commands.hx | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/qmp-commands.hx b/qmp-commands.hx
index 4942590..cedcd17 100644
--- a/qmp-commands.hx
+++ b/qmp-commands.hx
@@ -2576,6 +2576,12 @@ The main json-object contains the following:
 - "total-time": total amount of ms since migration started.  If
                 migration has ended, it returns the total migration
                 time (json-int)
+- "setup-time" amount of setup time in milliseconds _before_ the
+               iterations begin but _after_ the QMP command is issued.
+               This is designed to provide an accounting of any activities
+               (such as RDMA pinning) which may be expensive, but do not 
+               actually occur during the iterative migration rounds 
+               themselves. (json-int)
 - "downtime": only present when migration has finished correctly
               total amount in ms for downtime that happened (json-int)
 - "expected-downtime": only present while migration is active
@@ -2629,6 +2635,7 @@ Examples:
           "remaining":123,
           "total":246,
           "total-time":12345,
+          "setup-time":12345,
           "downtime":12345,
           "duplicate":123,
           "normal":123,
@@ -2653,6 +2660,7 @@ Examples:
             "remaining":123,
             "total":246,
             "total-time":12345,
+            "setup-time":12345,
             "expected-downtime":12345,
             "duplicate":123,
             "normal":123,
@@ -2672,6 +2680,7 @@ Examples:
             "remaining":1053304,
             "transferred":3720,
             "total-time":12345,
+            "setup-time":12345,
             "expected-downtime":12345,
             "duplicate":123,
             "normal":123,
@@ -2697,6 +2706,7 @@ Examples:
             "remaining":1053304,
             "transferred":3720,
             "total-time":12345,
+            "setup-time":12345,
             "expected-downtime":12345,
             "duplicate":10,
             "normal":3333,
-- 
1.7.11.7

